src/Home.tsx

```diff
@@ -78,12 +86,11 @@ const HomePage: React.FC = () => {
               ))}
             </View>
             <View style={styles.buttonsContainer}>
-              <TouchableOpacity style={styles.button}>
-                <Text style={styles.buttonText}>Recipes</Text>
-              </TouchableOpacity>
-              <TouchableOpacity style={styles.button}>
-                <Text style={styles.buttonText}>Consult with chatGPT</Text>
-              </TouchableOpacity>
+              {buttons.map((button, index) => (
```

Comment:
instead of 
`{buttons.map((button, index) => (`
do
`{buttons.map(({label, onPress}, index) => (`

Reply:
and then also change the relevant code

Reply:
done